rancher/distros-test-framework
# feat: add Aws sdk

#### Proposed Changes ####



- Add Aws sdk abstraction methods for create and delete instances 
- Refactor and improvements related



#### Types of Changes ####



#### Testing ####


Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.

- n/a
2. Verify code lint; we should not have errors.

- ok
3. Update the documentation if needed.

- ok 
4. Update makefile and docker run if adding new tests.

-n/a
5. Run your tests at least 4 times with all configurations needed and possible.
- ok 

6. If needed test with different os types.

-n/a
#### Linked Issues ####



#### Further Comments ####
